src/applications/find-forms/widgets/createFindVaFormsPDFDownloadHelper/DownloadPDFModal.jsx

```diff
@@ -0,0 +1,76 @@
+// Dependencies.
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Modal from '@department-of-veterans-affairs/component-library/Modal';
+
+// DownloadPDFModal is state wrapper + modal for PDF guidance upon PDf being valid
+const DownloadPDFModal = ({ formNumber, url }) => {
+  const [modalState, setModalState] = useState({
+    isOpen: true,
+    pdfSelected: formNumber,
+    pdfUrl: url,
+  });
+
+  const toggleModalState = () => setModalState({ isOpen: !modalState.isOpen });
```

Comment:
```suggestion
  const toggleModalState = () => setModalState({ ...modalState, isOpen: !modalState.isOpen });
```
I think we would want to maintain the rest of the object?